translations/harbour-contrac-de.ts

```diff
@@ -306,60 +305,60 @@
     </message>
     <message id="contrac-settings_he_server_details">
         <source>Server details</source>
-        <translation type="unfinished"></translation>
+        <translation>Server-Details</translation>
     </message>
     <message id="contrac-settings_co_download-coverage">
         <source>Download coverage</source>
-        <translation type="unfinished"></translation>
+        <translation>Download-Abdeckung</translation>
```

Comment:
"Abdeckung klingt furchtbar und hat imho überhaupt keine Sinn in dem Kontext. Linguee schlägt hier unter anderem Umfang vor, was meiner Meinung nach ganz gut passt. Vielliecht fällt die aber noch ein besserer Begriff, ganz passend finde ich "Download-Umfang" auch nicht. https://www.linguee.de/deutsch-englisch/search?source=auto&query=coverage

Reply:
"Abdeckung" ist ein gebräuchlicher Terminus aus Navigation und Telekommunikation ("es werden 20 Länder abgedeckt"/"inklusive Kartenabdeckung für Europa und Nordamerika"), der im Kontext von Contrac denke ich auch Sinn macht. Den Begriff "Download" habe ich vom Original übernommen, wie würde dir dann alternativ "Länderabdeckung" taugen?

Reply:
Länderabdeckung klingt besser :+1: